
The usual type conflict cought by gcc >= 3.4.x.

  - Rene Rebe <rene@exactcode.de>

diff -ur elvis-tiny-1.4.orig/config.h elvis-tiny-1.4.orig-patched/config.h
--- elvis-tiny-1.4.orig/config.h	1991-08-04 23:20:32.000000000 +0200
+++ elvis-tiny-1.4.orig-patched/config.h	2005-02-02 19:22:42.021908344 +0100
@@ -178,13 +178,6 @@
 # define uchar		unsigned char
 #endif
 
-/* Some compilers prefer to have malloc declared as returning a (void *) */
-#if BSD
-extern void *malloc();
-#else
-extern char *malloc();
-#endif
-
 /* Most compilers could benefit from using the "register" storage class */
 #if 1
 # define REG	register
